test/NaCl.Core.Benchmarks/XSalsa20Benchmark .cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace NaCl.Core.Benchmarks
```

Comment:
We could switch to scoped namespace declaration here since we already using it on the other files =)